core/domain/feedback_services.py

```diff
@@ -801,6 +804,19 @@ def _send_feedback_thread_status_change_emails(
                 old_status, new_status)
 
 
+def _check_unique_ids_for_recipients(user_ids, exploration_id, thread_id):
+    user_models = (
+        email_models.FeedbackEmailReplyToIDModel.get_for_multi_user_ids(
```

Comment:
The get_multi_for_user_ids() function implements get_multi() in back. Since we don't have what will be structure of keys in this function, get_multi() can not be used.